appengine/deploy/build-and-deploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+
+PROJECT_ID=$1
+
+echo "Reading configs from cloud secret manager"
+gcloud --project=${PROJECT_ID} secrets versions access latest --secret="study-manager-config" > vault.conf
```

Comment:
vault integration via files? 😸

Reply:
Vault?   What's  that?  :)

Check  out the  [secrets here](https://console.cloud.google.com/security/secret-manager?folder=&organizationId=&project=broad-ddp-dev)